open-sphere-base/analysis/src/main/java/io/opensphere/analysis/table/renderers/StringTableCellRenderer.java

```diff
@@ -47,7 +49,14 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
         if (value != null)
         {
             String sVal = value.toString();
-            if (myAutoFitRowHeight && sVal.length() > 50)
+            if (myListener.hasURLFor(row, column, sVal))

```

Comment:
we have a lot of null/blank values in the data, so it'd be nice to add that into the 'if' to avoid having to call the linker logic